spec/features/user_login_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+feature "User try sign in to the page" do
+  scenario "when successfull" do
+    login_as create(:user)
+
+    expect(current_path).to eq books_path
+    expect(page).to have_link 'Logout'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.